### PR TITLE
Fix Widgets.image()

### PR DESCRIPTION
**Pull request by SweetSea-ButImNotSweet**

### CODE REVIEW

**Comment by MrZ626** [comment]
I'm worried that this make image widget too complex...
I have an idea to simplify your additions a bit:
remove `keepAspectRatio`
if both `w` & `h` exist, use both of them
if non of them exist, use original size
if only one of them exist, scale the image to match it

File: widget.lua [hunk 1]
```diff
@@ -1012,6 +1038,7 @@ function Widgets.slider_fill:reset()
     self._rangeR=self.axis[2]
     self._rangeWidth=self._rangeR-self._rangeL
     self._pos=self._rangeL
+    self._pos0=self._rangeL
```
Comment (MrZ626) [inline widget.lua:1041]: why

**Comment by MrZ626** [comment]
btw I finished WIDGET.image's scaling

File: widget.lua (hunk 1 above)
Comment (SweetSea-ButImNotSweet) [reply widget.lua:1041]: Idk, adding slider_fill at my side cause crash so I add that line for fix
Also I guess pos0 is the original value of pos so...

**Pull request closed without merging**